final_task/rss_reader/requirements.txt

```diff
@@ -0,0 +1,14 @@
+astroid==2.3.3
+beautifulsoup4==4.8.1
+bs4==0.0.1
+entrypoints==0.3
+isort==4.3.21
+lazy-object-proxy==1.4.3
+lxml==4.4.1
+mccabe==0.6.1
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pylint==2.4.3
+six==1.13.0
+soupsieve==1.9.5
+wrapt==1.11.2
```

Comment:
There are many libs that you do not use here. It is better to clean this up.

Reply:
fixed